docs/src/views/Colors.tsx

```diff
@@ -0,0 +1,352 @@
+import {
+  Provider,
+  ProviderConsumer,
+  Grid,
+  Header,
+  Box,
+  Text,
+  mergeThemes,
+  themes,
+} from '@stardust-ui/react'
+import * as _ from 'lodash'
+import * as React from 'react'
+import { Link } from 'react-router-dom'
+
+import ColorBox, { colorBoxStyles, colorBoxVariables } from 'docs/src/components/ColorBox'
+import Fader, { faderStyles } from 'docs/src/components/Fader'
+import ColorVariants, { colorVariantsStyles } from 'docs/src/components/ColorVariants'
+import DocPage from 'docs/src/components/DocPage/DocPage'
+import ExampleSnippet from '../components/ExampleSnippet'
+import ColorSchemes from 'docs/src/components/ColorSchemes'
+import GuidesNavigationFooter from 'docs/src/components/GuidesNavigationFooter'
+import { link, code } from 'docs/src/utils/helpers'
+
+const theme = {
+  componentStyles: {
+    Box: {
+      root: ({ theme: { siteVariables } }) => {
+        const colorScheme = siteVariables.colorScheme
+        return {
+          textAlign: 'center',
+          padding: '5px',
+          color: colorScheme.brand.foreground3,
+          backgroundColor: colorScheme.brand.background3,
+          border: `1px solid ${colorScheme.brand.border}`,
+          ':hover': {
+            color: colorScheme.brand.foregroundHover,
+            backgroundColor: colorScheme.brand.backgroundHover1,
+            border: `1px solid ${colorScheme.brand.borderHover}`,
+          },
+        }
+      },
+    },
+  },
+}
+
+const Colors = () => (
+  <Provider
+    theme={{
+      componentStyles: {
+        ColorBox: colorBoxStyles,
+        ColorVariants: colorVariantsStyles,
+        Fader: faderStyles,
+        Header: {
+          root: {
+            fontWeight: 700,
+          },
+        },
+        Text: {
+          root: ({ theme: { siteVariables } }) => ({
+            '& a': {
+              color: siteVariables.colorScheme.brand.foreground,
+            },
+          }),
+        },
+      },
+      componentVariables: {
+        ColorBox: colorBoxVariables,
+      },
+    }}
+  >
+    <ProviderConsumer
+      render={({
+        siteVariables: { colors, contextualColors, naturalColors, transparentColors },
+      }) => (
+        <DocPage title="Colors">
+          <Header as="h2">Introduction</Header>
+          <p>
+            The organizing of the colors for an application has many requirements and constraints.
+            There is a need to be intentional and functional with color use. We analyzed existing
+            frameworks and picked the best ideas from them. In Stardust, the colors mechanisms are
```

Comment:
> We analyzed existing frameworks and picked the best ideas from them.

We are great, but let's remove it

---

<pre>
The organizing of the colors for an application has many requirements and constraints. In Stardust, the colors mechanisms are completely based on the <code>siteVariables</code>. We have two concepts to work transparently when the theme switching is in play:
</pre>

My proposal to keep this in single paragraph

Reply:
👍